js/main.js

```diff
@@ -114,16 +129,22 @@ function sortPlots() {
     var selectedOptions = "<h3> x-axis: " + xAxis + "</h3>";
     selectedOptions += "<h3> y-axis: " + yAxis + "</h3>";
     for (var v = 0; v < filtersArr.length; v++){
+        if(filtersArr[v] == "justNull"){
```

Comment:
Change this conditional after you change the `value` in the option setup above.
Also, again, triple quality everywhere! Use `===` instead of `==`, also, flip this conditional and get rid of the continue, it simplifies it.

```javascript
if(filtersArr[v] !== "justNull"){
    selectedOptions += "<h3> " + filters[v] + ": " + filtersArr[v] + "</h3>";
}
```

Reply:
Done